saleor/graphql/checkout/tests/test_checkouts_query.py

```diff
@@ -0,0 +1,289 @@
+from decimal import Decimal
```

Comment:
To be consistent, this file should be moved and renamed to: `saleor/graphql/checkout/tests/queries/test_checkouts.py`

Reply:
To be consistent with what? Checkout doesn't have `queries`.  I would need to remove all query tests and place them inside `queries`.